webpki-root-certs/src/lib.rs

```diff
@@ -1 +1,330 @@
-//! Placeholder to replace with generated content
+//! A compiled-in copy of the full X.509 root certificates trusted by Mozilla.
```

Comment:
AIUI having this in a separate commit means tests would fail after the preceding commit.

Reply:
Squashed the generation + the addition of the test into the commit adding the generator.